src/devices/acurite.c

```diff
@@ -662,6 +731,106 @@ static int acurite_txr_decode(r_device *decoder, bitbuffer_t *bitbuffer)
             valid += acurite_6045_decode(decoder, bitbuffer, brow);
         }
 
+        else if (browlen == ACURITE_ATLAS_BITLEN / 8) {
```

Comment:
I *think* there are two different message lengths depending on whether the lightning detector is installed so this should handle both.

Ideally this code should be factored out into another function.

Reply:
This is true. With LD the message is 10 bytes, without, it is 8 bytes. We should check the message type rather than the message length I believe. We end up needing to check the type eventually anyway, we may as well do it early and save a step.